webapp/locales/en.json

```diff
@@ -102,7 +102,7 @@
       "projects": "Projects",
       "invites": "Invites",
       "follows": "Follows",
-      "shouts": "Shouts"
+      "shouts": "Recommended"
```

Comment:
I don't think we need to change this in English

Reply:
@mattwr18 do you think "Shouting" could be misunderstood in a negative way? As I pointed out in the comment above :point_up:?

Reply:
sorry, @roschaefer, I saw this, but didn't reply...
I don't think so, but maybe shout-out would be better... like to give someone or something a shout-out... just that I think Instagram already uses that